charts/portal/templates/configmap-backend-seeding-initialdata.yaml

```diff
@@ -0,0 +1,55 @@
+{{- /*
+* Copyright (c) 2024 Contributors to the Eclipse Foundation
+*
+* See the NOTICE file(s) distributed with this work for additional
+* information regarding copyright ownership.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Apache License, Version 2.0 which is available at
+* https://www.apache.org/licenses/LICENSE-2.0.
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+* License for the specific language governing permissions and limitations
+* under the License.
+*
+* SPDX-License-Identifier: Apache-2.0
+*/}}
+
+{{- if .Values.backend.portalmigrations.seeding.useInitial -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name:  "{{ include "portal.fullname" . }}-default-initialdata"
+  namespace: {{ .Release.Namespace }}
+data:
+  addresses.json: |-
+    [
+      {
+        "id": "b4db3945-19a7-4a50-97d6-e66e8dfd04fb",
+        "date_created": "2022-03-24 18:01:33.306000 +00:00",
+        "date_last_changed": "2022-03-24 18:01:33.306000 +00:00",
+        "city": "tbd",
+        "region": "",
+        "streetadditional": "",
+        "streetname": "tbd",
+        "streetnumber": "",
+        "zipcode": "",
```

Comment:
Please enable those values in the values.yaml file under operator.

Reply:
done